spec/lib/batch/add_publisher_for_collection_members_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/batch/add_publisher_for_collection_members"
+
+RSpec.describe 'rails runner lib/batch/add_publisher_for_collection_members.rb' do
+  before do
+    @user = FactoryGirl.create(:user)
```

Comment:
I don't expect you to re-write anything here, but we should absolutely move away from using member variables (I think that's what they call it when you use @) and instead try to use `let` blocks where we can. It's a pretty standard practice in newer rails apps and there are a few reasons for it that I won't get into here.

Reply:
Nice to know, I did not realize that. Please do continue to suggest changes, it is good to be aware of these things.